stable/provazio/templates/controller-namespace-admin-role-binding.yaml

```diff
@@ -1,26 +1,40 @@
 {{- if and (.Values.controller.enabled) (has "IguazioTenantAppServiceSet" .Values.controller.crd.kinds) }}
 
+# create a full list of managed namesapces (append current namespace with additionalManagedNamespaces)
+# done this way as a workaround for the helm bug, which translates {}->{""} when passing empty list on cli
+{{- $managedNamespaces := list }}
+{{- if and (eq (len .Values.additionalManagedNamespaces) 1) (has "" .Values.additionalManagedNamespaces) }}
+{{- $managedNamespaces = list .Release.Namespace }}
+{{- else }}
+{{- $managedNamespaces = append .Values.additionalManagedNamespaces .Release.Namespace }}
+{{- end }}
```

Comment:
why not just:
```
{{- $managedNamespaces = append (.Values.additionalManagedNamespaces .Release.Namespace) | uniq | sortAlpha }}
```

Reply:
look at the note above
```
# done this way as a workaround for the helm bug, which translates {}->{""} when passing empty list on cli
```
when list is empty, it'll be filled with {"", ""} and will try to create the same resource

Reply:
good idea, had to use `without` to remove the empty namespace from being there when an empty list is passed